### ISSUE TITLE
Issue with Select when using ->displayUsingLabels()

### ISSUE THREAD

**Issue by mitchkramez** [epartment/nova-dependency-container#87]
All selects outside of my NovaDependencyContainer work fine with the ->displayUsingLabels() method, however, those within a NovaDependencyContainer, won't render the selected value when editing, it shows as blank, and saves as such. Selecting a value and saving it, still works as expected though. Removing the ->displayUsingLabels() option seems to bring everything back to functional again.

```
public $bannerTypes = [
        'alert-danger' => 'Danger',
        'alert-warning' => 'Warning',
        'alert-info' => 'Info',
        'alert-success' => 'Success'
    ];

...

Boolean::make('Show Banner', 'has_banner')
    ->help('Show the banner for this plan.'),

NovaDependencyContainer::make([
    Text::make('Banner', 'banner_text')
        ->help('Alert Banner Text to display for this Plan. Leave blank to hide banner.')
        ->hideFromIndex(),
                
    Select::make('Banner Type')
        ->options($this->bannerTypes)
        ->displayUsingLabels()
        ->hideFromIndex()
        ->nullable(),
])->dependsOn('has_banner', 1),
```

Let me know if I can answer any questions you might have! Thanks for the great package 😀

**Comment by wize-wiz** [comment]
Sorry, iphone was being trigger frenzy closing this issue ..

**Comment by wize-wiz** [comment]
@mitchkramez Should be fixed in [1.2.8](https://github.com/epartment/nova-dependency-container/releases/tag/1.2.8), please confirm and if so, close this issue.

**Comment by mitchkramez** [comment]
@wize-wiz fully and completely fixed! thank you again!
